test/ttsafe_acreate.c

```diff
@@ -63,7 +63,11 @@ tts_acreate(void)
     int    buffer, i;
     herr_t status;
 
-    ttsafe_name_data_t *attrib_data;
+    ttsafe_name_data_t *attrib_data[NUM_THREADS];
+
+    char *attribute_name = NULL;
+
+    memset(attrib_data, 0, sizeof(attrib_data));
```

Comment:
Does this generate size warnings?

Reply:
Nothing cropped up in the -Werror builds. NUM_THREADS is 16, so this is 16 * 8 bytes per pointer = 128, which I think should be fine.

Reply:
The -Werror builds don't check --enable-threadsafe